lib/iex/lib/iex/config.ex

```diff
@@ -119,17 +119,7 @@ defmodule IEx.Config do
   # Agent API
 
   def start_link(_) do
-    Agent.start_link(__MODULE__, :handle_init, [@table], [name: @agent])
-  end
-
-  def new() do
-    tab = :ets.new(@table, [:named_table, :public])
-    true = :ets.insert_new(tab, [after_spawn: []])
-    tab
-  end
-
-  def delete(__MODULE__) do
-    :ets.delete(__MODULE__)
+    Agent.start_link(__MODULE__, :handle_init, [], [name: @agent])
```

Comment:
Nitpicky nitpick but I guess we tend to go with no `[]` on last argument kw list:

```elixir
Agent.start_link(__MODULE__, :handle_init, [], name: @agent)
```